src/Interpreters/Context.cpp

```diff
@@ -1558,8 +1549,6 @@ void Context::initializeSystemLogs()
     shared->system_logs.emplace(*global_context, getConfigRef());
 }
 
-<<<<<<< HEAD
-=======
```

Comment:
I would suggest to squash all commits and force push, to at least avoid code that does not compile in the repo 

@alexey-milovidov or even this is ok to? what is the policy?

Reply:
@azat Yes, we should squash if the history become too messy.
But I can do it directly on GitHub.